### DISCUSSION TITLE
Thanks Kuo Kuo for sharing this PicoBETH project!

### DISCUSSION THREAD

**Discussion by alberthsu0121** [206cc/PicoBETH#3] [Show and tell]
Thanks @206cc for sharing this PicoPeth Project!

My friend and I have successfully updated my AEF-5 (奧億 飛5) with PicoBETH!

![S__15269918](https://github.com/user-attachments/assets/35693c79-5139-4a6c-a9ea-8beb313f526c)

**Comment by ttoh** [comment]
@alberthsu0121 This is very cool. I am sourcing a similar slide table. What part did you use to attach the YZC-133 load sensor to the head of the slide table? Did you machine it yourself or is it available for purchase off-the-shelf? I have the same stringing machine and this would make assembly much easier.
